src/app/components/nav/header-static/header-static.component.ts

```diff
@@ -61,7 +60,7 @@ export class HeaderStaticComponent implements OnInit, OnDestroy {
   routePath = '/dashboard';
 
 
-  @ViewChild('navContainer') navContainer: ElementRef;
+  @ViewChild('navContainer', {read: null}) navContainer: ElementRef;
```

Comment:
Same here, please remove if not necessary.

Reply:
![Screen Shot 2020-01-06 at 22 35 51](https://user-images.githubusercontent.com/34530026/71828478-00ee3580-30d5-11ea-9549-e4bc91259c6a.png)
![Screen Shot 2020-01-06 at 22 36 27](https://user-images.githubusercontent.com/34530026/71828480-00ee3580-30d5-11ea-8753-eafca9d5218d.png)

It's necessary, if the opts parameter is not provided, the build will fail.

Reply:
It's a bit tricky with the static field, one version wants it, the other one rejects it. I follow the latter, the one that Travis is using.